### DISCUSSION TITLE
https://blog.3place.cn/3place/lmyn/LMYN036.html

### DISCUSSION THREAD

**Discussion by giscus** [hbaolong/hbaolong.github.io#83] [General]
# https://blog.3place.cn/3place/lmyn/LMYN036.html

https://blog.3place.cn/3place/lmyn/LMYN036.html

**Comment by picklecai** [comment]
时间过去了一百天年，

也是天上一日，人间一年的设定吗？

**Comment by picklecai** [comment]
将有很多的丈夫。

马善任人骑，地善任人踩呀！
